### PR TITLE
[IMP] l10n_in_withholding: remove withhold lines from withhold wizard

### PR DESCRIPTION
**Pull request by niai-odoo**
Before this commit:
In the Withhold wizard, multiple lines could be entered, allowing multiple tax lines in one withholding entry.

After this commit:
Multiple withhold lines have been removed from the wizard. The Withhold wizard now includes fields for the base amount, tax, and tax amount. Now, only one tax line is allowed for each withholding entry.

Task-4252179

### CODE REVIEW

File: addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.py [hunk 1]
```diff
@@ -68,14 +68,27 @@ def default_get(self, fields_list):
...
+        string="TDS Amount",
+        compute='_compute_amount',
+    )
+
+    #  ===== Constraints =====
+    @api.constrains('base', 'amount')
+    def _check_amounts(self):
+        for wizard in self:
+            precision = wizard.currency_id.decimal_places
+            if float_compare(wizard.base, 0.0, precision_digits=precision) <= 0:
```
Comment (jco-odoo) [inline addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.py:88]: currency.compare_amounts

File: addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml [hunk 2]
```diff
@@ -21,28 +21,21 @@
                 <sheet>
                     <group>
                         <group id="header_left_group">
-                            <field name="date"/>
-                            <field name="journal_id" domain="[('type', '=', 'general')]"/>
+                            <field name="date" string="Entry Date"/>
+                            <field name="base" widget="monetary" options="{'currency_field': 'currency_id'}" required="True"/>
+                            <field name="tax_id" domain="[('l10n_in_tds_tax_type', '=', l10n_in_tds_tax_type)]" required="True"/>
+                            <field name="amount"/>
```
Comment (jco-odoo) [inline addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml:27]: Why not monetary?  Or why do we need monetary in L25?

**Review by jco-odoo — COMMENTED** [review]
Minor stuff

File: addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml (hunk 2 above)
Comment (niai-odoo) [reply addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml:27]: Okay will make amount monetary.

**Comment by niai-odoo** [comment]
Hello @jco-odoo 
Thanks for the review
I have made the suggested changes

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by niai-odoo** [comment]
Hello, @jco-odoo 

I have resolved the test case and rebased the code. Can we move forward with this?

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**